src/modules/charge-versions/controllers/charge-version-workflow.js

```diff
@@ -41,13 +42,18 @@ const postChargeVersionWorkflow = async request => {
   const { licenceId } = request.payload;
   const { chargeVersion, user } = request.pre;
 
+  //  Create a tweaked version of the charge version object
+  //  This is because the 'status' at this point would not validate against the charge version model
+  const parsedCV = chargeVersion;
+  parsedCV.status = 'draft';
```

Comment:
do you have to declare a new variable or probably easier to simply add the status or update it i.e. 
chargeVersion.status = 'draft';

Reply:
You're right. It's not needed. I've changed this.